app.py

```diff
@@ -130,7 +130,7 @@ def webhook():
             send_message('Beep Boop. Village Bot shutting down...')
             return "ok", 200
         
-        elif userText.upper() == '/START' or userText.upper() == '/RUN' or userText.upper() == '/BOT' or userText.upper() == '/VILLAGEBOT':
+        elif userText.upper() in ['/START', '/RUN', '/BOT', '/VILLAGEBOT']:
```

Comment:
that's nice shorthand

Reply:
Since we moved from `==` to `in` I swapped the text match to before in in another commit